ztp/kube-compare-reference/metadata.yaml

```diff
@@ -204,5 +204,7 @@ fieldsToOmit:
       - pathToKey: metadata.labels."pod-security.kubernetes.io/warn-version"
       - pathToKey: metadata.annotations."capability.openshift.io/name"
       - pathToKey: metadata.annotations."olm.providedAPIs"
+      - pathToKey: metadata.annotations."operator.sriovnetwork.openshift.io/last-network-namespace"
+      - pathToKey: metadata.annotations."include.release.openshift.io/hypershift"
```

Comment:
Where does this annotation come in? Our current RDS configuration is not hypershift based so this should not be included here/ignored if found in the comparison.

Reply:
The "include.release.openshift.io/hypershift" is introduced on all 4.16 clusters seems like.
I checked two of my clusters(one installed with ABI, the other with ZTP), both of them have annotation "include.release.openshift.io/hypershift" in the cm collect-profiles-config. But on 4.14 clusters I don't see it.

Seems like added in this PR: https://github.com/openshift/operator-framework-olm/pull/697